### ISSUE TITLE
a11y - mat-option does not have a focus indicator when active in high contrast mode

### ISSUE THREAD

**Issue by architshukla** [angular/material2#10207]
#### Bug, feature request, or proposal:
Bug verified on Edge and FireFox on Windows.

#### What is the expected behavior?
When a mat-option is focused in high contrast mode, a focus indicator should denote the active option.

#### What is the current behavior?
There is no focus indicator for the active mat-option and so, there is no way to identify which option is active in high contrast mode.

#### What are the steps to reproduce?
1. On Windows, choose a High Contrast theme like "High Contrast 1"
2. Open this [Angular Docs example](https://material.angular.io/components/autocomplete/examples) or [StackBlitz example](https://angular-material2-issue-rkfhqc.stackblitz.io) in a browser that can render the web page in High Contrast.
3. Focus on the autocomplete field and press the down arrow. There is no visual indication for the active option.

#### What is the use-case or motivation for changing an existing behavior?
High contrast users do not have any visual indication of the active option.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 5.2.6
Angular Material 5.2.2
Windows 10
TypeScript 2.4.2
Browsers - Edge and IE

#### Is there anything else we should know?
Can be worked around by overriding styles and adding a transparent border around the active mat-option. I wasn't able to find the style of borders used to indicate focus in material. If there is a style guide for this, I can try a fix.
